packages/components/src/form/tests/chromatic/Form.chroma.jsx

```diff
@@ -88,7 +88,7 @@ stories()
                 </Field>
                 <Button variant="secondary">Submit</Button>
             </Form>
-            <Form disabled>
+            <Form disabled alignSelf="end">
```

Comment:
The align-self actually seems to move the form downward on desktop, what's the intention here?

Reply:
Without an align child of flex items stretch by default, making the smaller form stretch, start could be used too, iirc many chromatic tests behaves this way (aligning items at the end).

Reply:
Yeah but the rendering have been altered for a lg breakpoint, not sure this expected

Reply:
The goal is to make sure the forms don't stretch and keep their initial gap. Since the parent is a flex item it's needed I am not sure what you mean by rendering have been altered for a lg breakpoint compared to the old test?

Reply:
They are not vertically aligned because of the "alignSelf=end".

![image](https://user-images.githubusercontent.com/794579/154983280-11136d26-6e39-4cca-88f6-aba777c1a8d8.png)

I don't really mind, I didn't though it would escalade, I though it would be a "ah ah oopsy moment" and it would be over quickly. You can keep it this way if you want.

Reply:
Remove the align in the dev tools and you'll see what happens.

Reply:
![image](https://user-images.githubusercontent.com/794579/154994076-5ad41f68-3d6f-4f4e-a736-180e8cb0c518.png)